script/Ciel/Day 13/13_01_01_CIEL13_3.txt

```diff
@@ -197,52 +220,55 @@ I can't take my own life for the simple reason that I'm too afraid of dying.
 [sha:1254bcbc1df95caef1d3fb70601d85cfef5c1f21]{
 -- Page 111, Offset 33486.
 -- いつかどこかで、生き延びた以上、この命は限界まで使い切るのだと誓ったような―――
-I feel like... someday, somewhere, I made a pledge to live my life to the fullest until the very end before―――
+// I guess referencing offset 442? Nothing exact.
+I feel like... somewhere, long ago, I swore a vow that I'd live this life to the fullest―――
 }
 [sha:98b67ca1410c48dd18fedd92b8cad70c35e27c60]{
 -- Page 112, Offset 33487.
 -- 「……助け……誰かに、見張って、もらわないと……」
-"...Help... I need, someone to, watch over me..."
+"Help... I need... someone to... keep an eye on me..."
 }
 [sha:39b6d6b18ff41764c693f127526f037fb7206a52]{
 -- Page 112, Offset 33488.
 -- なら、せめて管理されればいい。
-If this is how it's going to be, then I should at least try to keep it under control.
+If I can't die, then I at least need to be kept under control.
 }
 [sha:98915e4d45da2d06554a00a89fefe2bd3e7b1a65]{
 -- Page 112, Offset 33489.
 -- 眠らないよう、俺が正気を失わないよう、誰かに見張って貰うしかない。
-I need to have someone watch over me, so that I don't fall asleep or lose my mind.
+Someone has to keep an eye on me to make sure I don't fall asleep or lose my mind.
 }
 [sha:c719d0668a3c1e6b379c818825fb093fa3136df6]{
 -- Page 113, Offset 33490.
 -- 「そうだ……先……輩」
-"That's right... Sen... pai."
+"That's right... Senpai..."
 }
 [sha:0fbea1ea6e3d6473fd4ee37f60afbf53284f7607]{
 -- Page 113, Offset 33491.
 -- こんな自分を見せたくなかったが、もうそんな余裕はない。
-I don't want her to see me as I am right now, but this is no time to be worrying about that.
+I don't want her to see me like this,
+ but now's not the time to be worried about appearances.
 }
 [sha:5d386acdfeada378b107bd7a7d9b9f1f85c34805]{
 -- Page 113, Offset 33492.
 -- それに、そうだ―――何かあったら相談していいと、先輩は言ってくれたじゃないか。
-And besides... that's right―――Senpai told me that I should turn to her if anything were to happen.
+Plus, now that I think about it――Senpai told me that I should turn to her if anything were to happen.
 }
 [sha:66ac8fdca848d194c57b07f2edbab09260e89913]{
 -- Page 114, Offset 33493.
 -- 最後の希望に縋ってポケットから携帯端末を取り出す。
-Latching on to my final hope, I take out my phone.
+Clutching at this last ray of hope, I take out my phone.
```

Comment:
I feel like "hold" is what you tend to use with these kinds of constructions? Though I do think simply "last hope" lends itself better then.
>Desperately holding onto this last hope